modules/tiled-gallery/tiled-gallery.php

```diff
@@ -228,6 +228,26 @@ static function get_talaveras() {
 		return self::$talaveras;
 	}
 
+	/**
+ 	 * Enqueue js for our jetpack/gallery block
+ 	 */
+ 	public static function enqueue_block_editor_assets() {
+  		wp_register_script(
+  			'jetpack-tiled-gallery-block',
+  			plugins_url( '../../_inc/build/modules-tiled-gallery-block.js', __FILE__ ), // this is built as a new webpack entry point
```

Comment:
Instead of `__FILE__` here we could use `JETPACK__PLUGIN_FILE` to avoid the `../../` relative paths.

Reply:
That's more robust, thanks. Now fixed :)